src/index.html

```diff
@@ -11,7 +11,111 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <!-- <script type="text/javascript" src="./main.js"></script> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit (https://mate-academy-form-lesson.herokuapp.com/create-application)">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          Surname:
+          <input type="text" name="surname" id="surname"
+            autocomplete="off" required>
+          <label for="surname"></label>
+        </div>
+        <div>
+          Name:
+          <input type="text" name="name" id="name"
+            autocomplete="off" required>
+          <label for="name"></label>
```

Comment:
value 'name' should be in label tag
```suggestion
          <label for="name">Name:</label>
```